client/state/comments/actions.js

```diff
@@ -0,0 +1,323 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import {
+	COMMENTS_RECEIVE,
+	COMMENTS_COUNT_RECEIVE,
+	COMMENTS_REQUEST,
+	COMMENTS_REQUEST_SUCCESS,
+	COMMENTS_REQUEST_FAILURE,
+	COMMENTS_REMOVE_COMMENT,
```

Comment:
Our action naming guidelines suggest that the verb is suffixed. https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#actions

So this can be shortened to `COMMENTS_REMOVE`